pkg/api/server/register_manifest4.go

```diff
@@ -0,0 +1,181 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/containers/podman/v3/pkg/api/handlers/libpod"
+	"github.com/gorilla/mux"
+)
+
+func (s *APIServer) registerManifest4Handlers(r *mux.Router) error {
+	// swagger:operation POST /libpod/manifests/create manifests ManifestCreateLibpod
+	// ---
+	// summary: Create
+	// description: |
+	//   Create a manifest list
+	//
+	//   As of v4.0.0
+	// produces:
+	// - application/json
+	// parameters:
+	// - in: query
+	//   name: name
+	//   type: string
+	//   description: manifest list or index name to create
+	//   required: true
+	// - in: query
+	//   name: image
+	//   type: string
+	//   description: |
+	//     One or more names of an image or a manifest list. Repeat parameter as needed.
+	// - in: query
+	//   name: all
+	//   type: boolean
+	//   description: add all contents if given list
+	// responses:
+	//   200:
+	//     schema:
+	//       $ref: "#/definitions/IDResponse"
+	//   400:
+	//     $ref: "#/responses/BadParamError"
+	//   404:
+	//     $ref: "#/responses/NoSuchImage"
+	//   500:
+	//     $ref: "#/responses/InternalError"
+	r.Handle("/v4.0.0/libpod/manifests/create", s.APIHandler(libpod.ManifestCreate)).Methods(http.MethodPost)
```

Comment:
Should we require that lists be created empty and then mutated to add images? Because I see a whole lot more options in the body parameter of ManifestAdd than I do here?

Reply:
@mheon Once I get PUT working, I'll add that body to CREATE so we no longer need the two round trips to create and set attributes.